packages/mint-components/src/components/tax-and-cash/sqm-tax-form/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,57 @@
+@owner:andy
+@author:andy
+
+
+Feature: Tax Document Submitted View
+        View that informs the user of their Tax Document submission status.
+
+	Background: A user has submitted their Tax Document
+		Given a user is on the Tax Document Submitted view
+
+	@motivating
+	Scenario: View Tax Document Submitted
+		Given the user has submitted a tax document
+		When they view Document Submitted
+		Then they should see the status of their Tax Document and Banking Information
+
+	@minutia
+	@ui
+	Scenario Outline: The Tax Form header is controlled via <documentType> prop
+	Given the <documentType>
+	Then the Tax Form header will display "<documentType> Tax Form"
+	Examples:
+		| documentType |
+		| W9 		   |
+		| W8-BEN       |
+		| W8-BEN-E     |
+
+	@minutia
+	@ui
+	Scenario Outline: The Tax Document Submitted badge view is controlled via props
+		Given the Tax Document is currently <status>
+		When they view the Tax Document Submitted
+		And they will see a badge with <status> text and a respective <badgeVariant>
```

Comment:
This step should be "Then" because it is the result of the previous "When" (why the tax document is submitten then they will see a badge).